modules/quadbin/snowflake/test/integration/QUADBIN_ISVALID.test.js

```diff
@@ -0,0 +1,24 @@
+const { runQuery } = require('../../../../../common/snowflake/test-utils');
+
+test('QUADBIN_ISVALID should detect valid indexes', async () => {
+    const query = 'SELECT QUADBIN_ISVALID(5209574053332910079) AS OUTPUT';
+    const rows = await runQuery(query);
+    expect(rows.length).toEqual(1);
+    expect(rows[0].OUTPUT).toEqual(true);
+});
+
+test('QUADBIN_ISVALID should detect invalid indexes', async () => {
+    const query = 'SELECT QUADBIN_ISVALID(1234) AS OUTPUT';
+    const rows = await runQuery(query);
+    expect(rows.length).toEqual(1);
+    expect(rows[0].OUTPUT).toEqual(false);
+});
+
+/*
```

Comment:
Why we don't include this test? If not included then the comment can be removed.

Reply:
Function fixed